cypress/.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
         bundledDependencies: false,
       },
     ],
+    'jest/valid-expect': 'off',
```

Comment:
Why is this needed?

Reply:
This is needed as BBD Assertions from Chai produce eslint errors. (https://docs.cypress.io/guides/references/assertions#BDD-Assertions)

![Capture d’écran 2021-05-17 à 23 05 00](https://user-images.githubusercontent.com/32459935/118556564-4fe5b980-b764-11eb-9b4c-b39197e6a6f0.png)

I wish I could find a cleaner solution.